README.md

```diff
@@ -56,6 +56,12 @@ To add a value into existing enum:
 add_enum_value :mood, "pensive"
 ```
 
+To remove a value from existing enum:
+
+```ruby
+remove_enum_value :mood, "pensive"
```

Comment:
This is a very dangerous operation.
We should warn users in docs that they have not used the value anywhere in database.

Reply:
> Make sure that value is not used anywhere in the database.

Will it work?

Reply:
Agree